packages/itwinui-css/src/input-container/input-container.scss

```diff
@@ -193,6 +193,14 @@
     &:is(:not(:first-child)) {
       padding-inline-start: var(--iui-size-2xs);
     }
+
+    &::placeholder {
+      --_iui-input-text-color: var(--iui-color-text-placeholder);
+      font-weight: var(--iui-font-weight-light);
+      opacity: 1;
+      user-select: none;
+      color: var(--_iui-input-text-color);
```

Comment:
why needed to add placeholder styling now?

Reply:
placeholder styling was adjusted in `iui-input` only. I'm not using that class here

Reply:
then does this change need to go in v2? for SearchBox

Reply:
right, yup

Reply:
#1516